src/Console/Commands/EventStoreWorker.php

```diff
@@ -76,31 +76,49 @@ private function connectToStream($streams, $callback): void
 
     private function processPersistentStream($eventStore, string $stream): void
     {
+        $onStart = LaravelEventStore::$onStart;
+        $onSuccess = LaravelEventStore::$onSuccess;
+        $onError = LaravelEventStore::$onError;
+        $onFinish = LaravelEventStore::$onFinish;
+
         $eventStore
             ->persistentSubscription($stream, config('eventstore.group'), $this->option('parallel') ?? 1)
-            ->subscribe(function (AcknowledgeableEventRecord $event) {
+            ->subscribe(function (AcknowledgeableEventRecord $event) use ($onStart, $onSuccess, $onError, $onFinish) {
+                $onStart($event);
                 try {
                     $this->dispatch($event);
                     $event->ack();
+                    $onSuccess($event);
```

Comment:
Instead of defining `$onStart` at the top and passing it to method, simply call it here per:

```
LaravelEventStore::$onStart($event);
```

Reply:
Updated :)